che-theia-plugins.yaml

```diff
@@ -1,5 +1,16 @@
 version: 1.0.0
 plugins:
+  - repository:
```

Comment:
The plugins part needs to be in a separate PR (the sidecar that would be generated not pushed to quay.io yet) see https://github.com/eclipse/che-plugin-registry/blob/master/CONTRIBUTE.md#contribution-flow

Reply:
You can actually merge them together if you really want to, but testing this PR will be difficult because the generated surge link won't contain a valid sidecar reference. :man_shrugging: